src/components/views/rooms/MessageComposer.tsx

```diff
@@ -666,6 +668,7 @@ export class MessageComposer extends React.Component<IProps, IState> {
         return (
             <Tooltip open={isTooltipOpen} description={formatTimeLeft(secondsLeft)} placement="bottom">
                 <div className={classes} ref={this.ref} role="region" aria-label={_t("a11y|message_composer")}>
+                    <HistoryVisibleBannerView vm={new HistoryVisibleBannerViewModel({ room: this.props.room })} />
```

Comment:
At every render of the `MessageComposer`, the vm will be recreated. We want to avoid that.
You can:
1. Create the vm in the message component (constructor/componentDidMount), put in the state, dispose the vm in componentWillUnMount
2. Create a functional component where you create the vm with https://github.com/element-hq/element-web/pull/31178 and render the view